src/main/java/org/scijava/console/ConsoleService.java

```diff
@@ -62,6 +63,12 @@
 	/** Notifies listeners of output sent to {@code stdout} or {@code stderr}. */
 	void notifyListeners(OutputEvent event);
 
+	/** Stream that writes to stdout without notifying listeners. */
+	PrintStream getSystemOutBypassListeners();
```

Comment:
Do you actively and acutely need this? Otherwise, I have serious reservations about including and maintaining this sort of short-circuiting.